### ISSUE TITLE
Identical setups produce different results

### ISSUE THREAD

**Issue by bradymadams** [tetonsim/is-cura-ui#99]
[Originally reported](https://trello.com/c/0O7clEY0/31-unexpectedly-receive-different-results-for-identical-setup) by @rickdalgarno 

[simple_lever.zip](https://github.com/tetonsim/is-cura-ui/files/4345578/simple_lever.zip)

The optimization results for the simple lever are non-deterministic. 

Build 84

Steps to reproduce:
1. Open the attached .stl
2. Setup the problem as shown below
3. Run a validation and check the computed values (often these are consistent)
4. Run a validation and note the computed values and the mod mesh and global infill values.
5. Close Cura
6. Repeat steps 1-5. The computed results and infill density values should be close, but not the same.

Attached is a debug .3mf up to the point where the Validation step has completed.

Also attached are screenshots of results from 2 different runs.

**Comment by bradymadams** [comment]
![setup](https://user-images.githubusercontent.com/23110600/76904675-0f3f9700-6866-11ea-8d96-ab1ab1046a30.png)
![image](https://user-images.githubusercontent.com/23110600/76904678-11a1f100-6866-11ea-874b-a8e40c3728e3.png)
![image (1)](https://user-images.githubusercontent.com/23110600/76904683-136bb480-6866-11ea-8830-6578220d171e.png)
![image (2)](https://user-images.githubusercontent.com/23110600/76904690-149ce180-6866-11ea-927c-0971843e87b2.png)
![image (3)](https://user-images.githubusercontent.com/23110600/76904694-1797d200-6866-11ea-84fd-eb6a888b82f5.png)

**Comment by bradymadams** [comment]
Just an additional note: I ran the 3MF with smartslice CLI and the results (printed to the terminal) always came back the same. So this is either a problem with the results assignment to the object that gets sent back to the plugin, or within the plugin itself.

**Comment by jake-kirkland-teton** [comment]
So after extensive testing, I have not been able to reproduce the issues involving the Optimizing results coming back inconsistently. I was able to get the Validation results to come back wildly inconsistently, however that appears to be an issue with Cura itself, because it has a bad habit of changing little details in the print settings on its own on shutdown and restart.

**Comment by jake-kirkland-teton** [comment]
I tested the Cura settings change issue without the SmartSlice plugin installed, and I was unable to reproduce the issue. Then of course I reinstalled the plugin, and still couldn't reproduce the issue. So since that is separate enough to be considered its own issue if need be, I am going to close this because I have been entirely unable to make the optimize step produce inconsistent results. If someone does come across this issue again, I will reopen and re-investigate using the new STL and setup.
